Removing AD instructions (#4996)
DataDog/integrations-core: openstack_controller/README.md

This check monitors [Openstack][1] from the controller node.

## Setup
### Installation

The Openstack_controller check is included in the [Datadog Agent][2] package, so you do not need to install anything else on your server.

### Configuration

> The openstack_controller integration is designed to collect information from all compute nodes and the servers running on them. This integration is designed to be run from a single Agent to monitor your openstack environment. This can be deployed on your controller node or an adjacent server that has access to the Keystone and Nova endpoints.

Reviewer: ```suggestion
The openstack_controller integration is designed to collect information from all compute nodes and the servers running it. The integration should be run from a single Agent to monitor your openstack environment, and can be deployed on your controller node or an adjacent server that has access to the Keystone and Nova endpoints.
```

   See the [sample openstack_controller.d/conf.yaml][2] for all available configuration options.

2. [Restart the Agent][3]

#### Containerized
For containerized environments, see the [Autodiscovery Integration Templates][4] for guidance on applying the parameters below.

| Parameter            | Value                                                                                                                                                                                       |
|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `<INTEGRATION_NAME>` | `openstack_controller`                                                                                                                                                                      |
| `<INIT_CONFIG>`      | blank or `{}`                                                                                                                                                                               |

> | `<INSTANCE_CONFIG>`  | `{"name": "<INSTANCE_NAME>", "keystone_server_url": "https://<KEYSTONE_SERVER_ENDPOINT>:<PORT>/","user": {"password": "<PASSWORD>", "name": "<USER_NAME>", "domain": {"id": "<DOMAINE_ID>"}}}` |

Reviewer 2: I'm not sure if this one makes sense as there's no autodiscover variables

> | `<INSTANCE_CONFIG>`  | `"name": "<INSTANCE_NAME>", "keystone_server_url": "https://<KEYSTONE_SERVER_ENDPOINT>:<PORT>/" "user":{"password": "<PASSWORD>", "name": "<USER_NAME>", "domain": {"id": "<DOMAINE_ID>"}}` |

Reviewer: ```suggestion
| `<INSTANCE_CONFIG>`  | `{"name": "<INSTANCE_NAME>", "keystone_server_url": "https://<KEYSTONE_SERVER_ENDPOINT>:<PORT>/","user": {"password": "<PASSWORD>", "name": "<USER_NAME>", "domain": {"id": "<DOMAINE_ID>"}}}` |
```
